org.tribler.Tribler.yaml

```
@@ -0,0 +1,64 @@
+id: org.tribler.Tribler
+command: tribler
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-download
```

```suggestion
  - --filesystem=xdg-download
```

The default download path Tribler suggests me is `/var/home/bpiotrowski/TriblerDownloads`.

---

Interesting, on Fedora and Ubuntu /home/<user>/Downloads/TriblerDownloads was selected. What could be an elegant solution? Possibly avoiding `--filesystem=home`?